src/internal/operators/onErrorResumeNext.ts

```diff
@@ -87,81 +80,53 @@ export function onErrorResumeNext<T, R>(array: ObservableInput<any>[]): Operator
  * @name onErrorResumeNext
  */
 
-export function onErrorResumeNext<T, R>(...nextSources: Array<ObservableInput<any> |
-                                                       Array<ObservableInput<any>>>): OperatorFunction<T, R> {
-  if (nextSources.length === 1 && isArray(nextSources[0])) {
-    nextSources = <Array<Observable<any>>>nextSources[0];
-  }
-
-  return (source: Observable<T>) => lift(source, new OnErrorResumeNextOperator<T, R>(nextSources));
-}
-
-/* tslint:disable:max-line-length */
-export function onErrorResumeNextStatic<R>(v: ObservableInput<R>): Observable<R>;
-export function onErrorResumeNextStatic<T2, T3, R>(v2: ObservableInput<T2>, v3: ObservableInput<T3>): Observable<R>;
-export function onErrorResumeNextStatic<T2, T3, T4, R>(v2: ObservableInput<T2>, v3: ObservableInput<T3>, v4: ObservableInput<T4>): Observable<R>;
-export function onErrorResumeNextStatic<T2, T3, T4, T5, R>(v2: ObservableInput<T2>, v3: ObservableInput<T3>, v4: ObservableInput<T4>, v5: ObservableInput<T5>): Observable<R>;
-export function onErrorResumeNextStatic<T2, T3, T4, T5, T6, R>(v2: ObservableInput<T2>, v3: ObservableInput<T3>, v4: ObservableInput<T4>, v5: ObservableInput<T5>, v6: ObservableInput<T6>): Observable<R>;
-
-export function onErrorResumeNextStatic<R>(...observables: Array<ObservableInput<any> | ((...values: Array<any>) => R)>): Observable<R>;
-export function onErrorResumeNextStatic<R>(array: ObservableInput<any>[]): Observable<R>;
-/* tslint:enable:max-line-length */
-
-export function onErrorResumeNextStatic<T, R>(...nextSources: Array<ObservableInput<any> |
-  Array<ObservableInput<any>> |
-  ((...values: Array<any>) => R)>): Observable<R> {
-  let source: ObservableInput<any> | null = null;
-
+export function onErrorResumeNext<T, R>(...nextSources: ObservableInput<any>[]): OperatorFunction<T, R> {
```

Comment:
suggestion: This doesn't relate to the changes made in this PR - it was pre-existing here and occurs elsewhere in the codebase, too - but I think the `R` type parameter should be removed and the return type should be `OperationFunction<T, unknown>`, 'cause that's what it is, here. The actual inferences are made in the overload signatures; the implementation has no idea what the output type should be. Type parameters that are not used serve no purpose and are inferred to be `unknown` anyway.

I think the uses of `any` within the implementation should also be replaced with `unknown`.

With those changes, there are not lint failures or compilation issues that I can see.